### PR TITLE
feat: add i18n

### PR DESCRIPTION
**Pull request by chiba233**
# Pull Request Template

## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**Does this PR require a change to the documentation?** (check one)

- [ ] Yes
- [x] No

If yes, please update the documentation accordingly.

**Does this PR require a change to the environment variables configuration?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the changes that need to be updated in the configuration (put your example environment variables in the code block below).

```bash

```

## Description

Fixes # (issue)

## How Has This Been Tested?

**Desktop:**

- OS:
  - [ ] Windows [version]
  - [x] macOS [version]
  - [ ] Linux [version]

**Smartphone:**

## Screenshots

Remove if no visual changes have been made.

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -5,3 +5,5 @@ NEXT_PUBLIC_API_URL=https://backend-api.testenv.w3itch.io
 
 # Support multiple chain id, split by comma e.g: 1,4,56,97
 NEXT_PUBLIC_CHAIN_ID=1,3,4,5,42,56,97
+
+NEXT_PUBLIC_INFURA_API_KEY=ffddca9dcfaa43338ad9d6e79940a50b
```
Comment (xiaotiandada) [inline .env:9]: 不需要提交 NEXT_PUBLIC_INFURA_API_KEY

File: .idea/.gitignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
```
Comment (xiaotiandada) [inline .idea/.gitignore:2]: 不需要提交 .idea

File: public/locales/en/common.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "Create an account on w3itch.io": "Create an account on w3itch.io"
```
Comment (xiaotiandada) [inline public/locales/en/common.json:2]: 后面需要整理一下 key 的格式

Comment (xiaotiandada) [reply public/locales/en/common.json:2]: i18n key 格式可以参考这里的吧 https://pro.ant.design/zh-CN/docs/i18n

File: i18next-parser.config.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
+// i18next-parser.config.js
+
+module.exports = {
```
Comment (xiaotiandada) [inline i18next-parser.config.js:3]: 如果没有修改配置 可以空，让其采用默认值

Comment (chiba233) [reply i18next-parser.config.js:3]: 有修改配置

File: public/locales/en/common.json (hunk 3 above)
Comment (chiba233) [reply public/locales/en/common.json:2]: 后面改，现在先push

File: i18next-parser.config.js (hunk 4 above)
Comment (xiaotiandada) [reply i18next-parser.config.js:3]: > 有修改配置

似乎修改了

defaultNamespace
useKeysAsDefaultValue

应该只用配置这些修改项即可，其他采用默人

**Pull request merged**